compiler/src/dmd/backend/code.d

```diff
@@ -85,15 +85,10 @@ union evc
 
 /********************** PUBLIC FUNCTIONS *******************/
 
-code *code_calloc();
-void code_free(code *);
-void code_term();
+public import dmd.backend.dcode : code_calloc, code_free, code_term, code_chunk_alloc, code_list;
```

Comment:
trailing ws

Reply:
Heh, forgot to turn the setting back on after https://github.com/libmir/mir-algorithm/pull/464#issuecomment-1532256617 :upside_down_face: